app/components/courses/summary_card_component.rb

```diff
@@ -24,37 +27,23 @@ def title
       end
     end
 
-    def location_key
-      t(".location_key.#{course.funding}", count: @available_placements_count)
-    end
-
     def location_value
-      return t('.location_value.not_listed') if @available_placements_count.zero?
+      return t('.location_value.not_listed') if course.available_placements_count.zero?
```

Comment:
I do wonder now this is the only placed used on available placements count.

```
Courses::Query.new(params: {}).scope.map.select { |c| c.available_placements_count.zero? }.blank?
=> true
```

No published courses that will ever evaluated to true. @Nitemaeric maybe we can remove this line and the `available_placements_count`.